### ISSUE TITLE
[EKS] [request]: No available versions of adot add on for EKS 1.30

### ISSUE THREAD

**Issue by sidick** [aws/containers-roadmap#2365]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
I would like there to be a version of the `adot` add on which is available for EKS 1.30

**Which service(s) is this request for?**
EKS

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
```
% aws eks describe-addon-versions --kubernetes-version 1.30 --addon-name adot
{
    "addons": []
}
```

**Are you currently working around this issue?**
By not upgrading to EKS 1.30

**Additional context**
Anything else we should know?

**Attachments**
If you think you might have additional information that you'd like to include via an attachment, please do - we'll take a look. (Remember to remove any personally-identifiable information.)

**Comment by sidick** [comment]
Going by https://github.com/aws-observability/helm-charts/pull/51 support is in the adot code, it just needs testing and releasing

**Comment by clementlecorre** [comment]
Any new?

**Comment by Tr-Weber** [comment]
We got also the issue now. Is there any news?

**Comment by bhanvendrasingh** [comment]
any update on it

**Comment by dejwsz** [comment]
that is funny as it is still not there, aws is giving new EKS version (so 1.30) without releasing all supported managed addons with it, should we just use the one from 1.29 (v0.94.1-eksbuild.1) and apply it by Terraform or AWS cli for the moment ?

**Comment by sidick** [comment]
> that is funny as it is still not there, aws is giving new EKS version (so 1.30) without releasing all supported managed addons with it, should we just use the one from 1.29 (v0.94.1-eksbuild.1) and apply it by Terraform or AWS cli for the moment ? 

I tried out of curiosity, the web interface doesn't even let you install the old version on a 1.30 cluster. If it was installed on 1.29 and upgraded it does continue to work though

**Comment by jnicholls** [comment]
@sidick That is interesting. I'll try it via API and see if it allows it or not, but I presume no.

**Comment by jnicholls** [comment]
> Going by [aws-observability/helm-charts#51](https://github.com/aws-observability/helm-charts/pull/51) support is in the adot code, it just needs testing and releasing

This technically is the CloudWatch observability agent, not ADOT. Two different things altogether afaik.

**Comment by jnicholls** [comment]
Thanks @mikestef9 for providing attention to this issue!

**Comment by BraymanGrip** [comment]
Any progress with supporting this?
We have 2 clusters, one had the ADOT addon installed pre-upgrade and one didn't have the addon.
The addon works as expected on the upgraded cluster, meaning there's no need for real changes to the addon itself, but just to upgrade the "supported" version to `1.30`.
It's almost 2 months since the release of EKS 1.30 and this addon still isn't supported.

**Comment by jnicholls** [comment]
@mikestef9 has marked it as being released soon, so I expect it this week. Is that fair @mikestef9?

**Comment by sonalita** [comment]
The same issue is also reported here.  https://github.com/aws/containers-roadmap/issues/2365

**Comment by nssol-zhujiafeng** [comment]
can aws give us a specific date when adot will be released？ I have 20 clusters in 1.30 and need to install adot

**Comment by bhanvendrasingh** [comment]
Did someone find any alternative workaround to work with adot with 1.30 EKS

**Comment by CihatDinc** [comment]
Any update on this? I need this.

**Comment by sidick** [comment]
There is now a version of the adot addon for EKS 1.30, it took a while but it made it, thanks :)

**Comment by mhausenblas** [comment]
Indeed, we released 1.30 support, please see https://aws-otel.github.io/docs/ReleaseBlogs/aws-distro-for-opentelemetry-eks-add-on-v0.102.0 for details.
